packages/bcp/package.json

```diff
@@ -1,8 +1,11 @@
 {
-  "name": "@cosmwasm/bcp",
+  "name": "@iov/cosmos-sdk",
```

Comment:
I assume the next step is to move this package out of the monorepo, right?

Reply:
Right. But before we can do that, we need to publish a version of @cosmjs/sdk38 to npm, which I'd do after #192